### PR TITLE
End of basic auth

### PR DESCRIPTION
**Pull request by rtirserio**
This removes the authentication we were doing and replaces it with something more in line with what production will do. 

To login, go to the login.html file and either choose a personal or enter credentials for a known user.

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -24,7 +23,8 @@ api.interceptors.response.use(response => response, (error) => {
       // exports of api to be undefined. So this causes an error for `userProfile.js` when
       // attempting to login. Went with the eslint quick re-enable to get around this.
       /* eslint-disable global-require */
-      require('./store').store.dispatch(authRequest(false));
+      // require('./store').store.dispatch(authRequest(false));
```
Comment (burgwyn) [inline src/api.js]: I left this comment in

File: public/script.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
 function login(username, password) {
-
   $.ajax({
     type:"POST",
-    url:"https://api.dev.talentmap.us/api/v1/accounts/token/",
+    url:"http://localhost:8000/api/v1/accounts/token/",
```
Comment (burgwyn) [inline public/script.js]: Could we make this something that runs through Webpack so we could dynamically assign routes?

File: public/script.js [hunk 3]
```diff
@@ -1,11 +1,10 @@
...
     data: "username=" + username + "&password=" + password,
     success: function(data) {
-      window.location.href = '/talentmap/tokenValidation?tmApiToken=' + data.token
+      window.location.href = '/tokenValidation?tmApiToken=' + data.token
```
Comment (burgwyn) [inline public/script.js]: See above comment

**Comment by rtirserio** [comment]
@burgwyn - in order to do the substitution using the env vars, the script file had to be put into the login.html file.

File: public/login.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
-
-<html>
+<!DOCTYPE html>
```
Comment (burgwyn) [inline public/login.html]: 👍

**Pull request merged**